pkgs/development/python-modules/pyxdg/default.nix

```diff
@@ -6,24 +6,16 @@
 
 buildPythonPackage rec {
   pname = "pyxdg";
-  version = "0.26";
+  version = "0.27";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "fe2928d3f532ed32b39c32a482b54136fe766d19936afc96c8f00645f9da1a06";
+    sha256 = "19f5j5mxp7ff0vp33s32qbpdi65iiwha0bj641gl70pdwnm97gc0";
   };
 
   # error: invalid command 'test'
   doCheck = false;
 
-  patches = [
-    # see: https://gitlab.freedesktop.org/xdg/pyxdg/-/merge_requests/5
-    (fetchpatch {
```

Comment:
Drop fetchpatch from the function signature, if you're removing it here.

Reply:
Done